### PR TITLE
Rename event messages for test.

### PR DESCRIPTION
**Pull request by alexander-aleksandrov**
We need to rename test messages of events so they will be more package specific. 
eg. `ProjectCreated` of the aggregate package to `AggProjectCreated`

### CODE REVIEW

**Comment by alexander-aleksandrov** [comment]
@armiol PTAL

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#Tue May 30 16:38:29 EEST 2017
+#Tue Jul 25 16:48:10 EEST 2017
```
Comment (armiol) [inline gradle/wrapper/gradle-wrapper.properties]: Not sure why this file is changed.

File: gradlew [hunk 2]
```diff
@@ -33,11 +33,11 @@ DEFAULT_JVM_OPTS=""
 # Use the maximum available, or set MAX_FD != -1 to use that value.
 MAX_FD="maximum"
 
-warn ( ) {
+warn () {
```
Comment (armiol) [inline gradlew]: Not sure why this file is changed.

File: server/src/test/java/io/spine/server/procman/ProcessManagerShould.java [hunk 3]
```diff
@@ -268,69 +270,69 @@ public void create_router() {
...
+        return ((PmAddTask.Builder) Sample.builderForType(PmAddTask.class))
                 .setProjectId(ID)
                 .build();
     }
 
     @SuppressWarnings("UnusedParameters") // OK for test class.
     private static class TestProcessManager extends ProcessManager<ProjectId,
-                                                                   Any,
-                                                                   AnyVBuilder> {
+            Any,
```
Comment (armiol) [inline server/src/test/java/io/spine/server/procman/ProcessManagerShould.java]: Please fix the formatting (make it aligned as previously).

**Pull request merged**